tests/test_audio_stream_handler_roi.py

```
@@ -1284,16 +1459,14 @@ async def post(self, url, data=None, files=None):
-            if url.endswith("/audio/transcribe"):
-                return SimpleNamespace(
-                    status_code=200,
-                    json=lambda: {"text": "Ile to pięć razy pięć?"},
-                    text="",
-                )
             return SimpleNamespace(
                 status_code=200,
                 json=lambda: {
                     "text": "25",
+                    "transcription": "Ile to pięć razy pięć?",
+                    "transcription_used_for_generation": "Ile to jest 5 × 5?",
```

Ruff (0.15.14)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)